lib/patch-series.ts

```diff
@@ -154,7 +155,7 @@ export class PatchSeries {
             basedOn, publishToRemote, baseCommit);
 
         return new PatchSeries(notes, options, project, metadata,
-            rangeDiff, coverLetter, senderName);
+            rangeDiff, md2text(coverLetter, 76), senderName);
```

Comment:
`76` what? Looks like you're just moving this "magic number" down the road.

nit: avoid magic numbers

Reply:
It's the `columns` parameter of the `md2text()` function...

Reply:
Yeah, but why `76`?

Reply:
To clarify (it was already explained in the commit message, but that was not obvious from the diff), I introduced a constant.